src/plugins/highlight blocks.py

```
@@ -0,0 +1,213 @@
+
+    while filenotcreated == True:
+        try:
+            attempt += 1
+            os.makedirs("HoloprintPluginHighlightBlocks" + str(attempt))
+            filenotcreated = False
+        except FileExistsError:
+            filenotcreated = True
+
+    os.makedirs(f"HoloprintPluginHighlightBlocks{str(attempt)}/textures/holoprint/entity")
+
+    with open(f"HoloprintPluginHighlightBlocks{str(attempt)}/manifest.json", "x") as file:
```

Ruff (0.16.1)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)